### PR TITLE
Translate task summary to chinese 

### PR DESCRIPTION
**Pull request by jiaqiw09**
# What does this PR do?

Part of  #26803 

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [x] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests? _not necessary_

## Who can review?
@stevhliu

### CODE REVIEW

**Comment by jiaqiw09** [comment]
@stevhliu  

here is another pr. it may result in some merge conflict problem as i have another PR is still under review. 

You can deal with it later.

Best

File: docs/source/zh/task_summary.md [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+      'token': 1761,
+      'token_str': ' platform',
+      'sequence': 'Hugging Face is a community-based open-source platform for machine learning.'}]
+    ```
+
+## 多模态
+
+多模态任务要求模型处理多种数据模态（文本、图像、音频、视频）以解决特定问题。图像描述是一个多模态任务的例子，其中模型将图像作为输入并输出描述图像或图像某些属性的文本序列。
+
+虽然多模态模型处理不同的数据类型或模态，但内部预处理步骤帮助模型将所有数据类型转换为`embeddings`（`vectors or list of numbers`，包含有关数据的有意义信息）。对于像图像描述这样的任务，模型学习图像嵌入和文本嵌入之间的关系。
```
Comment (stevhliu) [inline docs/source/zh/task_summary.md:324]: Should "vectors or list of numbers" also be translated?

```suggestion
虽然多模态模型处理不同的数据类型或模态，但内部预处理步骤帮助模型将所有数据类型转换为 embeddings（vectors or list of numbers，包含有关数据的有意义信息）。对于像图像描述这样的任务，模型学习图像嵌入和文本嵌入之间的关系。
```

File: docs/source/zh/task_summary.md [hunk 2]
```diff
@@ -0,0 +1,347 @@
...
+>>> from transformers import pipeline
+
+>>> transcriber = pipeline(task="automatic-speech-recognition", model="openai/whisper-small")
+>>> transcriber("https://huggingface.co/datasets/Narsil/asr_dummy/resolve/main/mlk.flac")
+{'text': ' I have a dream that one day this nation will rise up and live out the true meaning of its creed.'}
+```
+
+## 计算机视觉
+
+计算机视觉任务中最早成功之一是使用卷积神经网络（CNN）识别邮政编码数字图像。图像由像素组成，每个像素都有一个数值。这使得将图像表示为像素值矩阵变得容易。每个像素值组合描述了图像的颜色。
```
Comment (stevhliu) [inline docs/source/zh/task_summary.md:67]:
```suggestion
计算机视觉任务中最早成功之一是使用卷积神经网络（[CNN](glossary#convolution)）识别邮政编码数字图像。图像由像素组成，每个像素都有一个数值。这使得将图像表示为像素值矩阵变得容易。每个像素值组合描述了图像的颜色。
```

**Review by stevhliu — COMMENTED** [review]
Thanks, remember to add this doc to the `toctree`! 🙂

**Comment by jiaqiw09** [comment]
@stevhliu 

hi, thansk for your review. I have fix the reviews and conflict problem. 

Best

**Comment by jiaqiw09** [comment]
It seems there is still some problem with yml file. I will fix it later

File: docs/source/zh/_toctree.yml [hunk 3]
```diff
@@ -30,3 +30,7 @@
   - local: serialization
     title: 导出为 ONNX
   title: 开发者指南
+- sections:
+  - local: 任务综述
```
Comment (stevhliu) [inline docs/source/zh/_toctree.yml:34]:
```suggestion
  - local: task_summary
```

**Comment by jiaqiw09** [comment]
@stevhliu

hi, thansk for your review.  I have fixed yml problem.

Best

**Comment by jiaqiw09** [comment]
@stevhliu

hi, thansk for your review.  I have fixed yml problem.

Best

**Review by stevhliu — APPROVED** [review]
Thanks again! 👍

**Pull request merged**